parm/gdas/soil/regrid.nml_tmpl

```diff
@@ -0,0 +1,26 @@
+&config
+  n_vars=@[n_vars],
+  variable_list=@[soil_incr_vars]
+  missing_value=0.,
+@[HIDE_LIAU] n_tims=@[n_tims],
+@[HIDE_LIAU] time_list=@[time_list]  
```

Comment:
Instead of hiding in the namelist, why not provide the default values.  The fortran code will default to those, no?